### ISSUE TITLE
Provide a general summary of the issue

### ISSUE THREAD

**Issue by imabot2** [twbs/bootstrap#39385]
### Prerequisites

- [X] I have [searched](https://github.com/twbs/bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed issues
- [X] I have [validated](https://html5.validator.nu/) any HTML to avoid common problems
- [X] I have read the [contributing guidelines](https://github.com/twbs/bootstrap/blob/main/.github/CONTRIBUTING.md)

### Describe the issue

Consider using multiple collapse element with the `parent` option : 

```
const bsCollapse1 = new bootstrap.Collapse('#collapse1', {
  toggle: false, 
  parent: parent
})
```

When the second collapsible element is opened while the first one is still transitioning, the two collapsible elements can be open at the same time.

### Reduced test cases

The following CodePen shows the problem 

[https://codepen.io/Fifi-Lulu/pen/xxMdqre](https://codepen.io/Fifi-Lulu/pen/xxMdqre)

The two collapsible elements should never be opened at the same time.

### What operating system(s) are you seeing the problem on?

Linux

### What browser(s) are you seeing the problem on?

Chrome

### What version of Bootstrap are you using?

5.3.2

**Comment by julien-deramond** [comment]
Thanks for reporting this issue @imabot2 
I've just checked your CodePen example. Is what you're trying to achieve is something like the following?

```js
const parent = document.getElementById('parent')
const bsCollapse1 = new bootstrap.Collapse('#collapse1', {
  toggle: false, 
  parent: parent
})
const bsCollapse2 = new bootstrap.Collapse('#collapse2', {
  toggle: false, 
  parent: parent
})

bsCollapse1.show();

document.getElementById('collapse1').addEventListener('shown.bs.collapse', event => {
  bsCollapse2.show();
});
```

Because in the doc we mention the following, but I'm not sure if it's related to your use case exactly:

> `show`: Shows a collapsible element. Returns to the caller before the collapsible element has actually been shown (e.g., before the shown.bs.collapse event occurs).

**Comment by imabot2** [comment]
@julien-deramond No, this is not what I want to achieve. I reported a general issue which, I think is clear :

> When the second collapsible element is opened while the first one is still transitioning, the two collapsible elements can be opened at the same time. The two collapsible elements should never be opened at the same time.

**Comment by MerlinChatou** [comment]
I've got exactly the same problem.

**Comment by mdo** [comment]
In v6-dev, accordion items are native `<details>` sharing a `name` attribute, so the browser closes the others the instant one opens — two siblings in the same group can't be open simultaneously (verified even under same-tick/forced toggling). The JS collapse race this describes no longer applies to the v6 accordion. Closing as resolved in v6-dev — please open a new issue if you can still reproduce it there.
